nav2_bringup/README.md

```diff
@@ -1,6 +1,15 @@
 # nav2_bringup
 
-The `nav2_bringup` package is an example bringup system for Nav2 applications.
+The `nav2_bringup` package is an example bringup system for Nav2 applications. 
+
+This is a very flexible example for nav2 bring up that can be modified for different maps/robots/hardware/worlds/etc. It is our expectation for an application specific thing you're mirroring `nav2_bringup` package and modifying it for your specific  maps/robots/bringup needs. So this is an applied and working demonstration for the default system bringup with many options that can be easily modified. 
+
+Usual robot stacks will have a `<robot_name>_nav` package with config/bringup files and this is that for the general case to base a specific robot system off of.
+
+Composed bringup (based on  [ROS2 Composition](https://docs.ros.org/en/galactic/Tutorials/Composition.html) ) is optional for user, in other word, compose all Nav2 nodes in a single process instead of launching these nodes separately, which is useful for embedded systems users that need to make optimizations due to harsh resource constraints.  
```

Comment:
```suggestion
Composed bringup (based on  [ROS2 Composition](https://docs.ros.org/en/galactic/Tutorials/Composition.html)) is optional for users. It can be used to compose all Nav2 nodes in a single process instead of launching these nodes separately, which is useful for embedded systems users that need to make optimizations due to harsh resource constraints.  
```